lib/llm/src/model_card.rs

```diff
@@ -108,6 +119,10 @@ pub struct ModelDeploymentCard {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub chat_template_file: Option<PromptFormatterArtifact>,
 
+    /// Custom chat template provided via CLI flag
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub custom_chat_template: Option<PromptFormatterArtifact>,
```

Comment:
Wouldn't it make sense to put this in `chat_template_file` instead of a new field? Will the frontend ever want both at once?